web_src/css/editor/combomarkdowneditor.css

```diff
@@ -37,13 +37,13 @@
 .combo-markdown-editor textarea.markdown-text-editor {
   display: block;
   width: 100%;
-  min-height: 200px;
-  max-height: calc(100vh - 200px);
+  min-height: 140px;
```

Comment:
looks like a magic number, maybe a css-var is easier to maintain and with the name of the var everybody knowns what the number means

Reply:
Added a per-element variable in https://github.com/go-gitea/gitea/pull/29232/commits/59230d28c940a088d5e9814ef42e92c0312a12b4. Normally I do not recommend such variables because they complicate the themeing and I think all theme-related vars should be on `:root`, but it does make a certain sense to have "private" variables on elements like in this case.

Reply:
Actually I think a global variable makes sense here, added in https://github.com/go-gitea/gitea/pull/29232/commits/ec7e0539e3b9a0747efd2be0d44a49902af7c8e4. The current 132px fits exactly 6 lines on my screen:

<img width="72" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/e0fd87a3-95a6-455a-a593-e6d1b58b6164">